packages/shopify-codemod/test/fixtures/assert/falsy-called-method-to-assert-not-called/basic.output.js

```diff
@@ -0,0 +1,2 @@
+assert.neverCalledWith(foo, bar, qux);
```

Comment:
If it doesn't appear in our code, then don't worry about it (I think the example you put is probably the best translation).